tests/helpers/perf_run_helper.rb

```diff
@@ -255,6 +257,33 @@ def puppet_infrastructure_tune_current
     return output
   end
 
+  # Restart the pe-puppetserver service
+  #
+  # @author Bill Claytor
+  #
+  # @return [void]
+  #
+  # @example
+  #   restart_puppetserver
+  #
+  def restart_puppetserver
```

Comment:
This method is never used as far as I can tell.  Bill switched over to using master_manipulator.  So this may be an artifact the time lap between Justins original PR and now...

Reply:
Method deleted